robot_skills/src/robot_skills/torso.py

```diff
@@ -24,12 +24,16 @@ def __init__(self, robot_name, tf_listener, get_joint_states):
         super(Torso, self).__init__(robot_name=robot_name, tf_listener=tf_listener)
 
         self.joint_names = self.load_param('skills/torso/joint_names')
-        self._arm_joint_names = self.load_param('skills/arm/joint_names')
         self.default_configurations = self.load_param('skills/torso/default_configurations')
         self.default_tolerance = self.load_param('/skills/torso/default_tolerance')
         self.lower_limit = self.default_configurations['lower_limit']
         self.upper_limit = self.default_configurations['upper_limit']
 
+        try:  # TODO make sure this is no longer needed
```

Comment:
Why not load them outside and provide them to the torso via the constructor?

This solution isn't very nice because of two reasons:
* the try-except is error-prone. I'm not sure what'll happen if you move the torso
* it makes the torso depend on the fact that your arm is called 'arm_center', which is specific to Hero. So while making the Hero object more generic by removing 'left', you make this more specific by adding 'arm_center'

Reply:
agreed, but this is a hotfix for hero anyway. The only change made here is that the dependency on some arm is made explicit.
A different PR should make this parameter no longer necessary in the first place. So that we can send only one joint goal while keeping all others the same.

Reply:
Can I see that :blush: ? I like keeping the master branch as clean as possible and these hotfixes do not contribute to that.

Reply:
You are right. I will make this cleaner. Probably by allowing arm joint names to be passed in construction, rather than in the part itself